src/components/UploadModal/UploadModal.scss

```diff
@@ -161,6 +161,14 @@
       }
     }
 
+    .modal-support-message {
+      @include typographyRegularXxs;
+      size: 10px;
```

Comment:
what does this do?

Reply:
![image](https://user-images.githubusercontent.com/4383992/206803027-90f60580-885c-4269-b69e-e8f6d4b3126b.png)
Just a message for notifying user that currently it's only arxiv.org

Reply:
I know that, but I'm talking about the  `size: 10px` line, I don't think that's valid css

Reply:
will remove

Reply:
is this ok to merge now?

Reply:
yes, but we have new conflicts now :sweat_smile: